clean.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree
+
+set -eu
+rm -rf cmake-out pip-out
```

Comment:
Could also add `cmake-android-out` and `cmake-out-android-*`

Reply:
actually not creating sh.clean.sh anymore. 

Reusing `install_requirements.sh --clean` instead.